tests/integration/targets/aci_vrf_leak_internal_subnet/tasks/main.yml

```diff
@@ -0,0 +1,183 @@
+# Test code for the ACI modules
+# Copyright: (c) 2023, Abraham Mughal (@abmughal)
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI APIC host, ACI username and ACI password
+  fail:
+    msg: 'Please define the following variables: aci_hostname, aci_username and aci_password.'
+  when: aci_hostname is not defined or aci_username is not defined or aci_password is not defined
+
+- name: Set vars
+  set_fact:
+    aci_info: &aci_info
+      host: '{{ aci_hostname }}'
+      username: '{{ aci_username }}'
+      password: '{{ aci_password }}'
+      validate_certs: '{{ aci_validate_certs | default(false) }}'
+      use_ssl: '{{ aci_use_ssl | default(true) }}'
+      use_proxy: '{{ aci_use_proxy | default(true) }}'
+      output_level: '{{ aci_output_level | default("debug") }}'
+
```

Comment:
cloud sites are failing, think we can skip those because the feature is not supported: "msg": "APIC Error 1: Invalid Configuration CLOUD_ROUTE_MAP_BASED_ROUTE_LEAKING_NOT_SUPPORTED: Route-maps based route leaking is not supported uni/tn-ansible_test3/ctx-test/leakroutes",

If not supported add cloud check and skip, see example https://github.com/CiscoDevNet/ansible-aci/blob/master/tests/integration/targets/aci_access_port_block_to_access_port/tasks/main.yml#L12C47-L12C47

Reply:
Same applies for the cn-dmz-apic-m1-02-v42 node which should be excluded from testing: "msg": "APIC Error 12: Unknown class leakTo"